index.js

```diff
@@ -0,0 +1,67 @@
+module.exports = {
+  "rules": {
+    "at-rule-empty-line-before": [ 2, "always", {
+      except: ["blockless-group"],
+    } ],
+    "at-rule-no-vendor-prefix": 0,
```

Comment:
This is actually a good idea. But I don't know if the comment added to a lib is a good idea. Especially if people are using the lib but not styelint. The comment should perhaps not understandable. what do you think ?